.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/assets/ansible-role-mysql"]
```

Comment:
It'd go for git subtree or put the synching logic into the pytest setup. Making the contributors do this is fragile.